crates/binstalk-downloader/Cargo.toml

```diff
@@ -43,7 +43,7 @@ xz2 = "0.1.7"
 # Since zip 0.6.3 depends on zstd 0.11, we can use 0.12.0 here
 # because it uses the same zstd-sys version.
 # Otherwise there will be a link conflict.
-zstd = { version = "0.12.2", default-features = false }
+zstd = { version = "0.12.3", default-features = false }
```

Comment:
0.12.3 added new feature `fat-lto` and `thin-lto`, though it also requires the linker to support the lto flavor (gcc vs llvm) it uses.

I will attempt to enable this in a separate PR.